packages/parts/src/lib/settings/WhereSpoken.svelte

```diff
@@ -43,19 +44,30 @@
   {t ? $t('create.where_spoken') : 'Where is this language spoken?'}*
 </div>
 
+<div class="text-xs text-gray-600 mb-2">
+  {t ? $t('create.map_instructions') : 'Click on the map to add secondary coordinates.'}
+</div>
 {#if hasCoordinates}
   <div class="h-240px">
     <Map
       lng={dictionary.coordinates.longitude}
       lat={dictionary.coordinates.latitude}
       on:click={({ detail }) => (mapClickCoordinates = detail)}>
+      <NavigationControl />
       {#if mapClickCoordinates}
         <CoordinatesModal
           {t}
           lng={+mapClickCoordinates.lng.toFixed(4)}
           lat={+mapClickCoordinates.lat.toFixed(4)}
           on:update={addCoordinates}
-          on:close={() => (mapClickCoordinates = null)} />
+          on:close={() => (mapClickCoordinates = null)}>
+          <Marker
+            lng={dictionary.coordinates.longitude}
+            lat={dictionary.coordinates.latitude}
+            color="blue">
+            <Popup offset={30} open={true}>Primary coordinate</Popup>
```

Comment:
Depending on context, sometimes I want the popup to start opened, sometimes not.